src/api/team-resolvers.ts

```diff
@@ -0,0 +1,71 @@
+import { MongoClient } from "mongodb";
+import { QueryResolvers } from "./graphql-types/__generated__/graphql-types";
+import { TeamMemberDocument } from "../storage/types";
+
+interface GenerateTeamResolversInput {
+  mongoClient: MongoClient;
+}
+
+interface GenerateTeamResolversOutput {
+  teamResolvers: { Query: QueryResolvers }
+}
+
+export const generateTeamResolvers = (input: GenerateTeamResolversInput): GenerateTeamResolversOutput => {
+  const { mongoClient } = input;
+
+  const databaseName = process.env.DATABASE_NAME;
+
+  if(!databaseName) {
+    throw new Error("Unable to find value for DATABASE_NAME. Please make sure you have run generate-env-files.sh and have specified one in the appropriate environment file.")
+  }
+
+  const groupedTeamMembers = async () => {
+    const groupedTeamMemberData = await mongoClient.db(databaseName).collection<TeamMemberDocument>('teamMembers').aggregate([
```

Comment:
https://www.mongodb.com/docs/manual/reference/method/db.collection.aggregate/

Output looks like the schema of the graphql resolver.